Adapter/Product/AdminProductDataProvider.php

```diff
@@ -293,12 +295,14 @@ public function getCatalogProductList($offset, $limit, $orderBy, $sortOrder, $po
         // post treatment
         $currency = new \CurrencyCore(\Configuration::get('PS_CURRENCY_DEFAULT'));
         foreach ($products as &$product) {
-            $product['price'] = \ToolsCore::displayPrice($product['price'], $currency);
             $product['total'] = $total; // total product count (filtered)
             $product['price_final'] = \ProductCore::getPriceStatic($product['id_product'], true, null,
-                    (int)\Configuration::get('PS_PRICE_DISPLAY_PRECISION'), null, false, true, 1,
-                    true, null, null, null, $nothing, true, true);
-            $product['price_final'] = \ToolsCore::displayPrice($product['price_final'], $currency);
+                (int)\Configuration::get('PS_PRICE_DISPLAY_PRECISION'), null, false, true, 1,
+                true, null, null, null, $nothing, true, true);
```

Comment:
$nothing is undefined. Mandatory due to legacy code that we cannot change for our Adapter needs.